apps/studio/src/features/editing-experience/components/EditSettingsPreview.tsx

```diff
@@ -110,6 +184,44 @@ export const EditSettingsPreview = ({
         theme={theme}
         callback={jumpToFooter ? handleIframeMount : undefined}
       >
+        {showChromeTab && (
+          <Box
+            style={{
+              display: "flex",
+            }}
+          >
+            <WindowButtons />
+            <ChromeTab
+              style={{
+                marginLeft: "-24px",
+                marginTop: "8px",
+              }}
+            >
```

Comment:
question: since ChromeTab only used here, why not put these margins into the component directly?

Reply:
these props don't feel liek they're part of the component but more dictated by the parent component

Reply:
but this component got reworked (no longer my handrolled styling lol) so this is no longer applicable